comps/llms/text-generation/ollama/llm.py

```diff
@@ -49,4 +49,5 @@ async def stream_generator():
 
 if __name__ == "__main__":
     ollama_endpoint = os.getenv("OLLAMA_ENDPOINT", "http://localhost:11434")
+    model_name = os.getenv("OLLAMA_MODEL", "meta-llama/Meta-Llama-3-8B-Instruct")
```

Comment:
Why meta-llama/Meta-Llama-3-8B-Instruct?

Reply:
It's default value. In case OLLAMA_MODEL is not set, then it is 'meta-llama/Meta-Llama-3-8B-Instruct'

Reply:
model=input.model if input.model else model_name